api/presenters/UserPresenter.js

```diff
@@ -25,6 +25,16 @@ const recentTaggedPost = (userId, tag, viewingUserId) => {
   .then(post => post && PostPresenter.present(post, viewingUserId, opts))
 }
 
+var sharedCommunities = (user, viewingUserId) => Promise.join(
```

Comment:
we have `Membership.sharedCommunityIds` already which does this more efficiently, in a single query